### PR TITLE
Feat/four stage page

### PR DESCRIPTION
**Pull request by Parle648**
## Describe your changes

- Created four stage page: Routes inform block + Map with route

## Issue ticket code (and/or) and link

- [Link to JIRA ticket](https://codecraftersintership.atlassian.net/browse/SCRUM-45)

## Page

image

### **General**

- [x] Assigned myself to the PR
- [x] Assigned the appropriate labels to the PR
- [x] Assigned the appropriate reviewers to the PR
- [x] Updated the documentation
- [x] Performed a self-review of my code
- [x] Types for input and output parameters
- [x] Don't have "any" on my code
- [x] Used the try/catch pattern for error handling
- [x] Don't have magic numbers
- [x] Compare only with constants not with strings
- [x] No ternary operator inside the ternary operator
- [x] Don't have commented code
- [x] no links in the code, env links should be in env file (for example: server url), constant links (for example default avatar URL) should be in constant file.
- [x] Used camelCase for variables and functions
- [x] Date and time formats are on the constants
- [x] Functions are public only if it's used outside the class
- [x] No hardcoded values
- [x] covered by tests
- [x] Check your commit messages meet the [conventional commit format](https://www.conventionalcommits.org/en/v1.0.0/).

### Frontend

- [x] Components and business logic are separated
- [x] Colors, Font Size, and Font Name is on the theme or in the constants
- [x] No text in the components, use i18n approach
- [x] No inline styles
- [x] Imports are absolute
- [x] Attach a screenshot if PR has visual changes.

### CODE REVIEW

File: src/pages/CreateRoute/RouteManagement/components/Map/RoutingComponent.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+      return {
+        lat: parseFloat(data[0].lat),
+        lon: parseFloat(data[0].lon),
+      };
+    } else {
+      throw new Error('Address not found');
+    }
+  };
+
+  const handleCalculateRoute = async () => {
```
Comment (lyudad) [inline src/pages/CreateRoute/RouteManagement/components/Map/RoutingComponent.tsx:35]: после useEffect

File: src/pages/CreateRoute/RouteManagement/index.tsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+import { Box } from '@mui/material';
+
+import CreateRouteButtons from '@/pages/components/CreateRouteBtns';
+import CreateRouteProgressBar from '@/pages/components/CreateRouteProgressBar';
+import Map from './components/Map';
+import InformBlock from './components/InformBlock';
+import { mapBlockStyles } from './styles';
+
+const RouteManagementPage: FC = () => {
+  const locations = [
```
Comment (lyudad) [inline src/pages/CreateRoute/RouteManagement/index.tsx]: move out of the component

File: src/pages/CreateRoute/RouteManagement/index.tsx [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+              distance: 900,
+              id: 1,
+              orders: [
+                { time_range: '19:20 - 19:30', city: 'Berlin' },
+                { time_range: '19:20 - 19:30', city: 'Berlin' },
+                { time_range: '19:20 - 19:30', city: 'Berlin' },
+                { time_range: '19:20 - 19:30', city: 'Berlin' },
+              ],
+            },
+          ]}
```
Comment (lyudad) [inline src/pages/CreateRoute/RouteManagement/index.tsx:38]: left comment //TODO replace with real data

File: src/pages/CreateRoute/RouteManagement/components/Map/RoutingComponent.tsx (hunk 1 above)
Comment (Parle648) [reply src/pages/CreateRoute/RouteManagement/components/Map/RoutingComponent.tsx:35]: done

File: src/pages/CreateRoute/RouteManagement/index.tsx (hunk 2 above)
Comment (Parle648) [reply src/pages/CreateRoute/RouteManagement/index.tsx]: done

File: src/pages/CreateRoute/RouteManagement/index.tsx (hunk 3 above)
Comment (Parle648) [reply src/pages/CreateRoute/RouteManagement/index.tsx:38]: left todo comment

**Pull request merged**